RTech.CoffeeMachine.Api/appsettings.Development.json

```diff
@@ -4,6 +4,6 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
-  }
+  },
+  "WeatherProviderBaseAddress": "https://api.open-meteo.com"
```

Comment:
I consider this a "safe" value to commit to app settings (& therefore shared on other dev machines & visible in the repo).

If the value contained any secrets such as API keys would make this an unsuitable option & a local-only secrets file would be used instead.

A side note: both the `development` & `production` appsettings use the same url. If a staging url is beneficial, then ideally that would go into the `Development` appsetting file instead.